docs/src/main/asciidoc/logging.adoc

```diff
@@ -205,11 +198,11 @@ FINEST:: Increased debug output compared to `TRACE`, which might have a higher f
 |===
 
 
-== Runtime configuration
+== Configure the log level, category and format
 
 Runtime logging is configured in the `application.properties` file.
 
-Because JBoss Logging is built-in to Quarkus, link:https://quarkus.io/developer-joy/[unified configuration] is provided for all <<supported-logging-apis,supported logging APIs>>.
+Because JBoss Logging is built-in to Quarkus, link:https://quarkus.io/developer-joy/[unified configuration] is provided for all <<logging-apis,supported logging APIs>>.
```

Comment:
This is a nitpick but "built in" should not be hyphenated in this context because the compound adjective "built in" does not appear directly before a noun.

Reply:
fixed